tests/bluetooth/tester/src/btp_ccp.c

```diff
@@ -9,10 +9,36 @@
 #include "zephyr/sys/byteorder.h"
 #include <stdint.h>
 
+#include <../../subsys/bluetooth/audio/tbs_internal.h>
+
 #include <zephyr/logging/log.h>
 #define LOG_MODULE_NAME bttester_ccp
 LOG_MODULE_REGISTER(LOG_MODULE_NAME, CONFIG_BTTESTER_LOG_LEVEL);
 
+struct service_handles {
```

Comment:
Why can't we just reuse `btp_ccp_chrc_handles_ev`?

Reply:
Fixed